Prose/ProseLib/Sources/SidebarFeature/Rows/ActionButton.swift

```diff
@@ -27,9 +26,8 @@ struct ActionRow: View {
 
 struct ActionRow_Previews: PreviewProvider {
     static var previews: some View {
-        ActionRow(
+        ActionButton(
             title: "sidebar_groups_add",
```

Comment:
Oh, some not localized content which I didn't catch before! I'll fix it when I'll do the settings page (which is full of not localized content).

Reply:
That's fine in a preview, isn't it?

Reply:
Yes, but we should at least write some human-readable text (i.e. "Example"), this was intended to be localized (`LocalizedStringKey`).

Reply:
(But it's priority -1000, I'll just do it if some day I pass by doing some localization)

Reply:
Yeah, just fix it next time you look at that preview.